storage/schema/account.js

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
I can not find usages of this file maybe it's better to remove it now and re-introduce it when planed functionality is implemented? (Although we might want to merge this initial version in the feature branch and later on iterate over this details) thoughts?

Reply:
As this PR is not the final PR for the feature, let's leave the file here as reference.